src/applications/vaos/tests/e2e/va-appointment.cypress.spec.js

```diff
@@ -5,9 +5,7 @@ import {
 } from './vaos-cypress-helpers';
 import * as newApptTests from './vaos-cypress-schedule-appointment-helpers';
 
-describe('VAOS direct schedule flow', () => {
-  beforeEach(() => {});
-
+describe('VAOS direct schedule flow: flat facility page', () => {
```

Comment:
We could probably leave off the `: flat facility page` part of this name, it's basically the default flow now, with the two-step one being the variation.